utils/release_tracking.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import re
+import os
+import subprocess
+import sys
+
+
+def run_cargo_build(path):
+    print(f"cargo build, path: {path}")
+    command = f'cargo build'
+    subprocess.check_output(command.split(), cwd=path)
+
+
+def clone_repo(clone_dir, repo_name, branch):
+    git_repo = f"https://github.com/parallaxsecond/{repo_name}.git"
+    future_repo_dir = os.path.join(clone_dir, repo_name)
+    if not os.path.isdir(future_repo_dir):
+        command = f"git clone {git_repo} -b {branch} {future_repo_dir}"
+        subprocess.check_output(command.split())
```

Comment:
`check_output` requires a list, not a string.
As explained in a different comment, I first make a string literal for the command so that it's easier to read for us developers, and then I split it into a list for check_output. It's easier to read this way :)
I don't see how this is a security risk if all of the strings are hardcoded?

Reply:
After our discussion, I changed to shell=True